### PR TITLE
 Support experiments with multi-core fuzzers.

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Also support KLEE experiments without commenting out code.

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
@gtt1995 Once this lands I think I can run your experiment.

**Comment by gtt1995** [comment]
> @gtt1995 Once this lands I think I can run your experiment.

OK！Thanks.
What should I do next？

File: experiment/run_experiment.py [hunk 1]
```diff
@@ -239,6 +239,11 @@ def start_experiment(  # pylint: disable=too-many-arguments
     config['no_dictionaries'] = no_dictionaries
     config['oss_fuzz_corpus'] = oss_fuzz_corpus
     config['description'] = description
+    config['runner_machine_type'] = config.get('runner_machine_type',
+                                               'n1-standard-1')
+    config['runner_num_cpu_cores'] = config.get('runner_num_cpu_cores', 1)
+    # Note this is only used if runner_machine_type is None.
+    config['runner_memory'] = config.get('runner_memory', '12GB')
```
Comment (inferno-chromium) [inline experiment/run_experiment.py:246]: maybe move 12 gb to some constants file

**Review by inferno-chromium — APPROVED** [review]
Do you want to fix the klee fuzzer config in this cl as well ?

**Comment by jonathanmetzman** [comment]
> > @gtt1995 Once this lands I think I can run your experiment.
> 
> OK！Thanks.
> What should I do next？

Give me some time. Probably nothing.

**Comment by jonathanmetzman** [comment]
> Do you want to fix the klee fuzzer config in this cl as well ?

I think I did right :-)

File: experiment/run_experiment.py (hunk 1 above)
Comment (jonathanmetzman) [reply experiment/run_experiment.py:246]: 12 GB is just useful for klee. i'll mention that in a comment. I don't really think it should be a constant.
I'll probably just make this a comment.

**Pull request merged**